src/base-components/switch-base/index.js

```diff
@@ -1,41 +1,42 @@
-import React from 'react'
+import React, { forwardRef } from 'react'
 import PropTypes from 'prop-types'
 
 import './switch-base.css'
 
-const _switchClasses = () => ({
+const _switchClasses = () => Object.freeze({
```

Comment:
the purpose of freeze is so that these classes can be referenced directly without invoke a function